.github/workflows/validate.yml

```diff
@@ -29,3 +29,7 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: ./build.sh
+      - uses: JustinBeckwith/linkinator-action@v1
```

Comment:
@MeganSteenkamp Suggestion - use a script instead of relying on this action being maintained

Reply:
Perhaps the config could also be encapsulated in the script to avoid adding another config file to the root of the repo 👀

Reply:
Thanks @tom-blake I have now addressed this 🤠